force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -225,21 +219,24 @@ public with sharing class ClassesObjectsMethods {
      * This class can be used in the context of a library management system, bookstore, etc.
      * where you need to manipulate and manage book information.
      */
-    public class Book {
-        // Declare three public instance variables - title, author, pages
-        // title
-        // author
-        // pages
+    public class Book
+    {
+        // Declare public instance variables - title, author, pages
+        public String title;
+        public String author;
+        public Integer pages;
     
-        // Book constructor to initialize the title and author
-        public Book() {
-            // Initialize the title and author
+        // Book constructor to initialize the title and author. Don't initialize pages
+        public Book(String title, String author)
+        {
+            this.title = title;
+            this.author = author;
         }
-    
+
         // Method that returns the details of the book
-        public String getBookDetails() {
-            // return A string containing the details of the book in the format "Title: <title>, Author: <author>"
-            return null; // Replace null with the variable you used to store the result
+        public String getBookDetails()
+        {
+            return 'Title: ' + title + ', ' + 'Author: ' + author;
```

Comment:
```suggestion
            return 'Title: ' + this.title + ', ' + 'Author: ' + this.author;
```
Although not necessary in apex, the 'this' keyword can be used for best practice.